indexer/run.sh

```diff
@@ -98,26 +80,26 @@ index() {
 sed -i '/if package_exists snapcraft/,/fi/d' ./build/install-build-deps.sh
```

Comment:
we already installed those in `prepare.sh`, is it still needed? if so i'd drop the ones in `prepare.sh`.

Reply:
I think so. It doesn't add much latency (everything will be installed most of the time) but it makes sure if new dependencies this will make sure they are installed.